test/suites/common/setup.yaml

```diff
@@ -9,6 +9,15 @@ spec:
   params:
     - name: cluster-name
       description: Name of the cluster under test.
+    - name: kubernetes-version
+      description: Kubernetes version to deploy
+      default: "1.21"
```

Comment:
I'd hold off on enabling this configurability until we use it.

Reply:
I'd rather leave this. I've used it to test on my fork on different versions. It's pretty useful. :)